Common.props

```diff
@@ -6,8 +6,10 @@
         <OutDir>$(EnlistmentRoot)out\binaries\$(Configuration)\$(MSBuildProjectName)\</OutDir>
         <OutputPath>$(OutDir)</OutputPath>
 
-        <IntDir>$(EnlistmentRoot)out\intermediate\$(Configuration)\$(MSBuildProjectName)\</IntDir>
+        <IntDir>$(EnlistmentRoot)out\intermediate\$(MSBuildProjectName)\</IntDir>
```

Comment:
Won't this cause problems when building in the IDE for both Debug and Release without intervening clean?